### PR TITLE
Make oncokb-annotator compatible with Python 2 and Python 3

### LINKED ISSUE #31: Upgrade to python3
- Have a dependency file
- Add test coverage 
- Work with https://github.com/oncokb/oncokb-annotator/pull/21

Information related to this topic
- https://docs.python.org/3/howto/pyporting.html

### PR DESCRIPTION
**Pull request by victoria34**
Fix #31

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,2 +1,5 @@
+# for python2
 matplotlib==2.2.4
+# for python3
+# matplotlib==3.1.2
```
Comment (zhx828) [inline requirements.txt]: This is commented out.

File: README.md [hunk 2]
```diff
@@ -3,7 +3,7 @@
 
 # oncokb-annotator <a href="https://ascopubs.org/doi/full/10.1200/PO.17.00011"><img src="https://img.shields.io/badge/DOI-10.1200%2FPO.17.00011-1c75cd" /></a>  
 
-Annotates variants in MAF with OncoKB annotation.
+Annotates variants in MAF with OncoKB annotation. Supports both python2 and python3.
```
Comment (zhx828) [inline README.md]: I thought we cannot support both?

File: AnnotatorCore.py [hunk 3]
```diff
@@ -703,9 +693,9 @@ def plotclinicalactionability(annotatedclinicalfile, outfile, parameters):
             width = 0.75
             plts = [plt.bar(ind, levelcancerperc, width, color=levelcolors[level], bottom=accumlevelcancerperc)] + plts
             legends = [levellegend[level]] + legends
-            accumlevelcancerperc = map(sum, zip(accumlevelcancerperc,levelcancerperc))
+            accumlevelcancerperc = list(map(sum, zip(accumlevelcancerperc,levelcancerperc)))
 
-        ax = plt.axes()
+        ax = plt.gca()
```
Comment (zhx828) [inline AnnotatorCore.py]: Would this backward compatible?

File: requirements.txt (hunk 1 above)
Comment (victoria34) [reply requirements.txt]: For Python 2, we need to use matplotlib 2.2.4. For Python 3, we need to use matplotlib 3.1.2.

File: README.md (hunk 2 above)
Comment (victoria34) [reply README.md]: Sorry, I may not express the tile clearly. This PR makes oncokb-annotator supporting both python 2 and python 3.

File: AnnotatorCore.py (hunk 3 above)
Comment (victoria34) [reply AnnotatorCore.py]: I think so. I tested the function in python 2 and python 3. It works well.

**Comment by zhx828** [comment]
@victoria34 is there anything we need to do once we merge this pr? And did you finish the data comparison of MSK-IMPACT 2017?

**Comment by victoria34** [comment]
> @victoria34 is there anything we need to do once we merge this pr? And did you finish the data comparison of MSK-IMPACT 2017?

@zhx828 Nothing. I only updated codebase related to logging. I didn't change core functions. I have tested this PR on the MSK-IMPACT 2017 and the result is identical with the result generated based on the master branch.

**Comment by zhx828** [comment]
@victoria34 let's merge the pr when you are ready.

**Pull request merged**